app/nostra.rb

```diff
@@ -5,4 +5,47 @@
 require_relative './nostra/account.rb'
 
 module Nostra
+  def self.import_csv(csv_file, type = :one_money)
+    case type
+    when :one_money
+      add_by_csv(csv_file)
+    else
+      raise "#{type} csv not implemented for import!"
```

Comment:
By this you mean, instead of raising, just puts-ing a statement?

Reply:
yeah or returning a value.  i'm thinking, "why do i want this to raise an error?" "where/who is responsible for rescuing from this error?"

If this is used on a CLI, puts might be just fine here, if its an API, returning some value or a 400 or something is more useful to a client than raising an internal error, unless there's a plan to handle all the errors internally that you're creating.

I'm sure there's arguments both ways, but here's some blog i found about it https://rmosolgo.github.io/blog/2016/11/23/raising-exceptions-is-bad/

I think i've heard this opinion elsewhere.  Things like collecting errors in arrays like we did in the core gem, using things like promises that can always return a result, even if its a failure.  having something that returns is more reliable as a function, vs raising which can interrupt the normal flow control

Reply:
Great thoughts here. In this case, I think merely being informative is sufficient. I mean, I don't plan on supporting any other csv and really this is just a use case function for myself, but in the case that someone were attempting to import something else entirely, it might be useful to know. On the other hand, I wrote this to default to the accepted type and really one could plop anything else in there and it would be accepted as a `one_money` csv. So, this is all kind of redundant anyway. Which also brings up a potential future security consideration to make and so on. I think for now, I'm just going to drop the checking as I don't expect to pass any other type in. If it fails, it fails for the particular reason internally and things can be handled there. Thanks for your thoughts on this!